_health-care/_js/components/ContinueButton.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+/**
+ * A component for the continue button to navigate through panels of questions.
+ */
+class ContinueButton extends React.Component {
+  constructor() {
+    super();
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.props.onButtonClick();
+  }
+
+  render() {
+    return (
+      <div className="row progress-buttons">
+        <div className="small-8 columns">
+          <button
+              className="usa-button-primary"
+              id="continueButton"
+              onClick={this.handleClick}>Continue</button>
```

Comment:
Just pass `this.props.onButtonClick`?  No need to reroute through the current component class method.  Yay for functional programming (higher order functions specifically, but whatevs)!